nixos/modules/services/monitoring/netdata.nix

```diff
@@ -225,6 +253,29 @@ in {
     environment.etc."netdata/netdata.conf".source = configFile;
     environment.etc."netdata/conf.d".source = configDirectory;
 
+
+    systemd.tmpfiles.settings = lib.mkIf cfg.package.withNdsudo {
+      "95-netdata-ndsudo" = {
+        "/run/netdata/ndsudo/ndsudo" = {
+          "L+" = {
+            mode = "0755";
```

Comment:
Should this be maybe something like `0550` (`r-xr-x---`)?

Reply:
Got it to work with `0555` I guess more restrictive is better?